brave/browser-laptop
# Add Qwant.com as a search engine

Fixes https://github.com/brave/browser-laptop/issues/2701

Auditors: @bbondy

## Test Plan
1. Launch Brave and open Preferences
2. Set Qwant.com as your default search engine and enable suggestions
3. In the omnibox, type "what" and wait for results to show
4. Search in the omnibox, confirm it shows results on qwant.com
5. Try the search engine shortcut, :q and do a search in the omnibox
